hibernate-core/src/main/java/org/hibernate/metamodel/internal/MetamodelImpl.java

```diff
@@ -111,6 +111,8 @@
 
 	private final TypeConfiguration typeConfiguration;
 
+	private final transient Map<String, String[]> implementorsCache = new ConcurrentHashMap<>();
```

Comment:
I don't think we need to make it transient, the other similar maps such as `jpaEmbeddableTypeMap` are not. The entity graph one is a bit special.